File: product_code_2.py

def calcular_pago(producto, cantidad, precio_usdt):
    total_usdt = precio_usdt * cantidad
    tasa_eth = 3500  # Simulado
    total_eth = total_usdt / tasa_eth
    return f"Total a pagar: {total_usdt:.2f} USDT | {total_eth:.5f} ETH"

print(calcular_pago("Camiseta NFT", 2, 30))

File: product_code_2.js

function calcularPago(producto, cantidad, precioUSDT) {
    const totalUSDT = precioUSDT * cantidad;
    const tasaETH = 3500;  // Simulada
    const totalETH = totalUSDT / tasaETH;
    return `Total a pagar: ${totalUSDT.toFixed(2)} USDT | ${totalETH.toFixed(5)} ETH`;
}

console.log(calcularPago("Camiseta NFT", 2, 30));
